runner/templates/configmap.yaml

```diff
@@ -64,7 +64,8 @@ data:
       {{ if .Values.cache.enabled -}}
       [runners.cache]
         Type = {{ .Values.cache.type | quote }}
-        ServerAddress = "{{ template "minio.fullname" . }}:9000"
+        ServerAddress = "{{ template "minio.fullname" . }}-svc:9000"
```

Comment:
I'll move this `-svc` into the _helpers.tpl file then.
What do you think of including the port number in the minio.fullname template? This way the user can set the port inside `Values.cache.serverAddress`